app/payment-request-mapping/index.js

```diff
@@ -55,6 +60,15 @@ async function getSchemeCode (standardCode, transaction) {
   return schemeCode.schemeCode
 }
 
+async function getFrn (sbi, transaction) {
+  const frn = await db.frn.findOne({ where: { sbi: sbi } }, { transaction })
+  if (frn) {
+    return Number(frn.frn)
+  }
+
+  return null
```

Comment:
I would throw an error here for now rather than return null.  If there is no FRN match then we can't process the payment so no point doing anything with it.

Long term we'll have notifications outbound for failures but for now just throw an error.